docs/src/man/getting_started.md

```diff
@@ -159,14 +159,18 @@ using CSV
 using DataTables, CSV
 ```
 
-Datasets can now be read using
+Datasets can now be read from input sources using
```

Comment:
"input sources" is quite vague. Since this section talks about files, better say "files" instead to keep things simple. Reading from streams is a more advanced use case, which could be mentioned as a bonus.